tests/integration/questionnaire/test_questionnaire_list_collector.py

```diff
@@ -245,4 +245,4 @@ def test_cancel_text_displayed_on_remove_block_if_exists(self):
 
         self.get(remove_link)
 
-        self.assertInBody("Don’t need to remove this person?")
```

Comment:
The `test_cancel_text_displayed_on_remove_block_if_exists` test should be removed now we can no longer have cancel text on a remove page.

Reply:
Removed, thanks 👍